tests/Contracts/MockKubService.test.ts

```diff
@@ -0,0 +1,44 @@
+import { KubeServiceBase, KubeResourceType } from "../../src/Contracts/KubeServiceBase";
+
+class MockKubeService extends KubeServiceBase {
+    public fetch(resourceType: KubeResourceType): Promise<any> {
+        return Promise.resolve(resourceType);
+    }
+}
+
+describe("KubeServiceBase Tests", () => {
+    let service: MockKubeService;

```

Comment:
Shouldn't this be of type KubeServiceBase ?

Reply:
fixed